packages/jest-circus/src/legacy_code_todo_rewrite/jest-adapter-init.js

```diff
@@ -43,6 +43,32 @@ const initialize = ({
   global.fit = global.it.only;
   global.fdescribe = global.describe.only;
 
+  global.test.concurrent = (
```

Comment:
we could actually make a breaking change to `test.concurrent` to pass the context as first argument (rather than "done") like we were planning to do, that way we could make it work properly with state.